quiz-app/src/assets/translations/zh-cn.json

```diff
@@ -0,0 +1,2612 @@
+                    "questionText": "一个好的模型将有：",
+                    "answerOptions": [{
+                            "answerText": "大量的误报和真底片在其混乱矩阵",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "大量的真正的积极和真正的负面在其混乱矩阵",
+                            "isCorrect": "true"
+                        },
+                        {
+                            "answerText": "大量的真正反误矩阵",
+                            "isCorrect": "false"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": 17,
+            "title": "构建 Web 应用程序：讲座前测验",
+            "quiz": [{
+                    "questionText": "ONNX 代表什么？",
+                    "answerOptions": [{
+                            "answerText": "通过神经网络交换",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "开放神经网络交换",
+                            "isCorrect": "true"
+                        },
+                        {
+                            "answerText": "输出神经网络交换",
+                            "isCorrect": "false"
+                        }
+                    ]
+                },
+                {
+                    "questionText": "弗拉斯克是如何由它的创造者定义的？",
+                    "answerOptions": [{
+                            "answerText": "迷你框架",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "大框架",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "微型框架",
+                            "isCorrect": "true"
+                        }
+                    ]
+                },
+                {
+                    "questionText": "Python 的泡菜模块是做什么的",
+                    "answerOptions": [{
+                            "answerText": "序列化 Python 对象",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "去序列化 Python 对象",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "序列化和去序列化 Python 对象",
+                            "isCorrect": "true"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": 18,
+            "title": "构建 Web 应用程序：讲座后测验",
+            "quiz": [{
+                    "questionText": "我们可以使用哪些工具使用 Python 在网络上托管预先训练的模型？",
+                    "answerOptions": [{
+                            "answerText": "瓶",
+                            "isCorrect": "true"
+                        },
+                        {
+                            "answerText": "滕索弗.js",
```

Comment:
TensorFlow.js?

Reply:
Yeah, actually I am confused whether to change them or not. But I translate them. No problem at all.

Reply:
No one can understand.

Reply:
Let me correct typo and all.